### ISSUE TITLE
API changes in 9.95

### ISSUE THREAD

**Issue by Chunkywolf** [pauln/tailwind-home-assistant#5]
There were API changes in firmware version 9.95 that have broken local control and status until it is updated to a non-beta api key. Updated documentation was posted here: https://gotailwind.zendesk.com/hc/en-us/community/posts/5757663497869-Home-Assistant-status-not-accurate-after-9-95

**Comment by pauln** [comment]
Thanks for letting me know; I haven't updated to v9.95 yet so hadn't encountered the API changes.  I did reply to Scott's email to beta users to ask whether the security token had been implemented, but hadn't had a reply; I'll dig into that thread / the updated docs this evening and see what I can do.

**Comment by pauln** [comment]
I've just released v0.2.0 which adds support for the Local Control Key which was introduced in the V9.95 firmware.  You'll need to remove and re-add the integration once you've updated the component (via HACS or manually via custom_components, depending on how you installed it to begin with), and grab your Local Control Key from Tailwind Web - see the updated README in this repo for details.  I've also posted an update to the support thread you linked, to hopefully spread the word.
